proposals/description/fip_30b.txt

```diff
@@ -1 +1,19 @@
-text
\ No newline at end of file
+Summary:
+
+FEI DAO Timelock Upgrade
+
+Upgrades the FEI DAO Timelock and grants the guardian the power to veto proposals.
+Adds in functionality to roll back to the old timelock via DAO vote with a deadline of November 1st, 2021.
```

Comment:
Can take this line out of summary. Also the title is automatically appended from the json so can remove the title line as well